infra/testing/webdriver/getLastWindowHandle.js

```diff
@@ -1,52 +0,0 @@
-/*
-  Copyright 2019 Google LLC
-
-  Use of this source code is governed by an MIT-style
-  license that can be found in the LICENSE file or at
-  https://opensource.org/licenses/MIT.
-*/
-
-
-// Store local references of these globals.
-const {server, webdriver} = global.__workbox;
-
-const testServerOrigin = server.getAddress();
-
-/**
- * Gets the window handle of the last opened tab.
- *
- * @return {string}
- */
-const getLastWindowHandle = async () => {
```

Comment:
I don't remember why I added this, but just checking that you've confirmed it's no longer needed.

Reply:
Yeah, `TabManager` replaces this.